src/uct/base/uct_iface.c

```diff
@@ -334,6 +334,13 @@ UCS_CLASS_INIT_FUNC(uct_iface_t, uct_iface_ops_t *ops)
         self->ops.iface_fence = uct_base_iface_fence;
     }
 
+    if (ops->iface_progress_disable == NULL) {
+        self->ops.iface_progress_disable = (void *)ucs_empty_function_return_unsupported;
```

Comment:
better to set this to a default value (empty_function) in all transports, as we do today for other "missing" functionality